http-gateway/swagger.yaml

```diff
@@ -516,7 +530,7 @@ paths:
         - 'Resource Events'
       summary: 'Subscribe to events'
       description: |
-        Open a websocket stream and receive events from devices. The contents of filters in the SubscribeToEvents message determine which events will be sent to the stream.
+        Open a websocket stream and receive events from devices. The contents of filters in the SubscribeToEvents message determine which events will be sent to the stream. It will sents current events and then all new filtered events. To bypass current events, set  justNewEvents to true.
```

Comment:
gramarly

If you're interested in the current state right after subscribing, set the `includeCurrentState` query parameter to `true`.